R-packages/evalcast/R/get_predictions.R

```diff
@@ -126,7 +126,11 @@ get_predictions <- function(forecaster,
     return(preds)
   }
 
-  out <- rlang::inject(bettermc::mclapply(forecast_dates, get_predictions_single_date_, mc.cores = num_cores, !!!additional_mclapply_args)) %>% bind_rows()
+  if (parallel_execution) {
```

Comment:
issue: parallel_execution could be an integer as well, right?  Or have we overwritten it with a Boolean in this case?

Reply:
(think R does auto coerce to Boolean here but doesn't do what we want)